uint8 left_status
uint8 right_status
int16[4] e_data

